spec/modules-spec.js

```diff
@@ -129,3 +131,25 @@ describe('Module._nodeModulePaths', function () {
     })
   })
 })
+
+describe('require', () => {
+  describe('when loaded URL is not file: protocol', () => {
+    let w
+
+    beforeEach(() => {
+      w = new BrowserWindow({
+        show: false
+      })
+    })
+
+    it('searches for module under app directory', async () => {
+      w.loadURL('about:blank')
+      const result = await w.webContents.executeJavaScript('typeof require("q").when')
+      assert.equal(result, 'function')
+    })
+
+    afterEach(() => {
+      w.destroy()
```

Comment:
This should use `closeWindow` helper instead, see https://github.com/electron/electron/blob/efa28503a7e722a8dc217703618bf925c88d4a01/spec/api-browser-window-spec.js#L84-L86 for an example